File: sujet_machine/exemples/recette/skeleton/recette.pl
#!/usr/bin/perl

sub recette0{
  my($n, $ordre, $desordre) = @_;
  # Inserez votre code ici 
  
}
my $n = int( <STDIN> );
my $ordre = [ map { int($_) } split(/\s+/, <STDIN>) ];
my $desordre = [ map { int($_) } split(/\s+/, <STDIN>) ];
recette0($n, $ordre, $desordre);

File: sujet_machine/exemples/recette/skeleton/recette.rb
require "scanf.rb"
def recette0( n, ordre, desordre )
  # Inserez votre code ici 
  
end
n = STDIN.readline.to_i(10)
ordre = STDIN.readline.split(" ").map{ |x| x.to_i(10) }
desordre = STDIN.readline.split(" ").map{ |x| x.to_i(10) }
recette0(n, ordre, desordre)
